src/test/java/com/google/devtools/build/lib/runtime/BlazeVersionInfoTest.java

```diff
@@ -34,22 +33,22 @@
 
   @Test
   public void testEmptyVersionInfoMeansNotAvailable() {
-    BlazeVersionInfo info = new BlazeVersionInfo(Collections.<String, String>emptyMap());
+    BlazeVersionInfo info = new BlazeVersionInfo(ImmutableMap.of());
```

Comment:
it's typically good practice to not change unrelated code, as to keep a change as focused as possible. Feel free to send a separate clean up CL if you think this adds value :-).

Reply:
agree :)